CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
-## 0.11.8 (Next)
+## 0.11.9 (Next)
 
 ### Features
   * Your contribution here.
 
+## 0.11.8 (4 June, 2020)
```

Comment:
I should be releasing the new gem by the weekend. Please leave this without a date.

Reply:
@jlduran removed the date